lib/posix/posix.nim

```diff
@@ -1124,7 +1124,7 @@ var
     importc: "_CS_POSIX_V6_LPBIG_OFFBIG_LIBS", header: "<unistd.h>".}: cint
   CS_POSIX_V6_WIDTH_RESTRICTED_ENVS* {.
     importc: "_CS_POSIX_V6_WIDTH_RESTRICTED_ENVS", header: "<unistd.h>".}: cint
-  F_LOCK* {.importc: "F_LOCK", header: "<unistd.h>".}: cint
+  f_lock* {.importc: "F_LOCK", header: "<unistd.h>".}: cint
   F_TEST* {.importc: "F_TEST", header: "<unistd.h>".}: cint
```

Comment:
Sorry, had it a little turned around. Either way, if this is a go-ahead to do all the procs and variables in the file, I'll get on it.